go.mod

```diff
@@ -4,8 +4,8 @@ go 1.21
 
 require (
 	github.com/giantswarm/apiextensions-application v0.6.1
-	github.com/giantswarm/cluster-standup-teardown v1.0.3
-	github.com/giantswarm/clustertest v0.20.0
+	github.com/giantswarm/cluster-standup-teardown v1.1.1-0.20240513000831-b7c9d5dd7da3
```

Comment:
TODO: update after https://github.com/giantswarm/cluster-standup-teardown/pull/20 is merged and released.